app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def show_svg(path)
```

Comment:
How do we feel about this?

I just want to have my svgs inline.

Reply:
I like it!